PixelToolkit.py

```diff
@@ -16,14 +16,8 @@ def generate_password(length):
     args = parser.parse_args()
 
     if len(sys.argv) == 1:  # If no arguments passed -> GUI
-        from lib.gui import password_gen_top_level
-        import tkinter as tk
+        from lib.gui import main_window_generator
```

Comment:
Feel free to disagree: Why not just call it main_window?

Reply:
I just find it more descriptive, and I when I pass window as an argument for function that'll generate some content in it or above it i'll tend to name it main_window. It can be changed ofc

Reply:
"generator" sounds like this function should return something that was generated. But I don't have strong preference here. It can stay as it is